Rubberduck.Parsing/Grammar/VBA.g4

```diff
@@ -102,30 +25,30 @@ startRule : module EOF;
 
 module : 
 	WS?
-	endOfLine*
-	(moduleHeader endOfLine*)?
-	moduleConfig? endOfLine*
-	moduleAttributes? endOfLine*
-	moduleDeclarations? endOfLine*
-	moduleBody? endOfLine*
+	endOfStatement
```

Comment:
Ha! That's one thing I had noticed yesterday, but couldn't get the line-ending colon to work anyway... good to see I had *something* done right! :+1: